app/main/views/platform_admin.py

```diff
@@ -70,6 +70,29 @@ def platform_admin():
     )
 
 
+@main.route("/platform-admin/download-all-users")
+@user_is_platform_admin
+def download_all_users():
+
+    # Create a CSV string from the user data
+    csv_data = "Name,Email Address,Mobile Number,Service\n"
+    users = user_api_client.get_all_users_detailed()
+
+    if len(users) == 0:
+        return "No data to download."
+    users = json.loads(users)
+    for user in users:
+        if user["name"].startswith("e2e"):
+            continue
+        csv_data += f"{user['name']},{user['email_address']},{user['mobile_number']},{user['service']}\n"
```

Comment:
Sigh, good catch that is a legit use case.  I changed commas to tabs.

Reply:
this is how it's showing up now 
<img width="857" alt="image" src="https://github.com/user-attachments/assets/fd5a0a4c-4eb8-4fe6-b3aa-b487d45533ec">

Reply:
That is not the latest.  Can you check the code on your local?  It should have "\t" instead of ",".  I tested with "Comma, comma, and comma service" and it was fine (with \t).

Reply:
Columns resolved!